features/google/src/styles.css

```diff
@@ -1,4 +1,14 @@
 :root {
+    --background-dark: var(--color-dark);
+    --background-light: var(--color-white);
+
+    --text-white: var(--color-white);
+    --text-light: var(--color-white);
+    --text-dark: var(--color-indigo-950);
+
```

Comment:
I'd use the theme colors here and if you can't use the colors defined in `poly-look/src/css/colors.css` we are just duplicating colors that are already there.